src/index.ts

```diff
@@ -0,0 +1,22 @@
+export class Challenge
+{
+public buzz(value :number)
+{
+    return new Promise((resolve, reject) =>
+{
+    if ((value % 3 === 0) && (value % 5 === 0))
+    {
+      return resolve('fizzbuzz');
+    }
+    else if (value % 3 == 0)
```

Comment:
Triple equals should be used almost always for checks

This is a good article:
https://codeburst.io/javascript-double-equals-vs-triple-equals-61d4ce5a121a